### PR TITLE
[M1230] Bugs fixed

### PR DESCRIPTION
**Pull request by tienhoah**
- Project Name disappear when project info updates
- In sites and mrs, Count total in filter label not updating when trying to delete a record and then adding a new record

### CODE REVIEW

**Comment by tienhoah** [comment]
[Project Name and Filter label fixes](https://trello.com/c/xLaRI3vw/1230-project-name-and-filter-label-fixes)

File: src/app/project/controllers/managements.ctrl.js [hunk 1]
```diff
@@ -184,6 +184,12 @@ angular.module('app.project').controller('ManagementsCtrl', [
     };
 
     $scope.tableControl.recordsNotFiltered = function() {
+      if (
+        $scope.tableControl.records &&
+        $scope.tableControl.records.length > managementRecordsCount
```
Comment (saanobhaai) [inline src/app/project/controllers/managements.ctrl.js]: Should this be `$scope.tableControl.records.length != managementRecordsCount`? (Same for sites)

File: src/app/_mermaid_common/libs/directives/projectname.dir.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ angular.module('mermaid.libs').directive('projectname', [
         watchId: '@'
       },
       template:
-        '<div class="projectname hidden-xs" title="{{project_name_tooltip}}">{{project_name}}</div>',
+        '<div class="projectname" title="{{project_name_tooltip}}">{{project_name}}</div>',
```
Comment (saanobhaai) [inline src/app/_mermaid_common/libs/directives/projectname.dir.js]: This seems right, but was there some other reason this div started out hidden?

Comment (tienhoah) [reply src/app/_mermaid_common/libs/directives/projectname.dir.js]: I think the project name used to be right beside the mermaid icon in the top.

**Pull request merged**